web/tests/notifications.test.js

```diff
@@ -381,7 +381,7 @@ test("basic_notifications", () => {
 
     const message_1 = {
         id: 1000,
-        content: "@-mentions the user",
+        content: "@-mentions the user :smile:",
```

Comment:
Where does your test verify that this is handled correctly?

Reply:
I will make it actually check for the correct output.